Source/JavaScriptCore/assembler/MacroAssemblerMIPS.h

```diff
@@ -1358,6 +1357,12 @@ class MacroAssemblerMIPS : public AbstractMacroAssembler<Assembler> {
         }
     }
 
+    void loadPair32(const void* address, RegisterID dest1, RegisterID dest2)
```

Comment:
Do not add `const void*` one newly. Use `AbsoluteAddress`.

Reply:
done.